app/services/doc_auth/lexis_nexis/lexis_nexis_client.rb

```diff
@@ -15,17 +15,21 @@ def create_document
       def post_images(
         front_image:,
         back_image:,
+        selfie_image: nil,
         image_source: nil,
         user_uuid: nil,
-        uuid_prefix: nil
+        uuid_prefix: nil,
+        liveness_checking_enabled: nil
```

Comment:
Is there a reason why the default value here for `liveness_checking_enabled` is `nil`, while for `true_id_request.rb` below it's `false`?

Reply:
@night-jellyfish default to false.